app/scienceapi/static/admin/custom-event-admin.css

```diff
@@ -0,0 +1,17 @@
+/* Removes Timezone warning from admin */
+.timezonewarning {
+	display: none;
+}
+
+/* Remove relative datetime pickers */
+.datetimeshortcuts > * {
```

Comment:
I don't believe that's a problem; I'd guess that button simply won't ever be clicked (but others like 6PM, etc. might). We can add help text too, but I don't see a need to add more code to take something away that might give an easy hint.